docs/extensions.rst

```diff
@@ -213,3 +213,32 @@ Keys:
 
 The ASGI server will only send the trailing headers in case the client has sent the
 ``TE: trailers`` header in the request.
+
+Debug
+-----
+
+The debug extension allows a way to send debug information from the ASGI application to
+an entity acting as an ASGI server. This extension is not meant to be used in production,
+but only for testing purposes, and ASGI servers should not implement it.
+
+The entity acting as ASGI server will provide ``http.response.debug`` in the extensions
```

Comment:
I've added the `response` here, but I'm not sure if I should have...

The suggestions were `http.debug_info` and `http.test_info`, but I find redundant to add the word "info", since everything is info here... :eyes: I can change it tho...

Reply:
No, I think the new name is fine. I'd also remove "entity..." here and switch to "the ASGI context sent to the application"

Reply:
Like this? :thinking: 
```suggestion
The ASGI context sent to the application will provide ``http.response.debug`` in the extensions
```

Reply:
Yup!